<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageJobRunnerWidget</class>
 <widget class="QWidget" name="ImageJobRunnerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="optionsSection" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <layout class="QVBoxLayout" name="runnerSettingsBoxLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="candidateShapesPerStepLabel">
            <property name="text">
             <string>Candidate Shapes Per Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="candidateShapesPerStepSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="candidateShapesPerStepCountLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="mutationsPerCandidateShapeLabel">
            <property name="text">
             <string>Mutations Per Candidate Shape</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="mutationsPerCandidateShapeSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mutationsPerCandidateShapeCountLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="shapeOpacityLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Shape Opacity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="shapeOpacitySlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="shapeOpacityValueLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="maxThreadsLabel">
            <property name="text">
             <string>Max Threads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="maxThreadsSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="randomSeedLabel">
            <property name="text">
             <string>Random Seed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="randomSeedSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="usesEllipses">
        <property name="text">
         <string>Ellipses</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesQuadraticBeziers">
        <property name="text">
         <string>Quadratic Beziers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesRotatedRectangles">
        <property name="text">
         <string>Rotated Rectangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesRotatedEllipses">
        <property name="text">
         <string>Rotated Ellipses</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesTriangles">
        <property name="text">
         <string>Triangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesPolylines">
        <property name="text">
         <string>Polylines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesCircles">
        <property name="text">
         <string>Circles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesRectangles">
        <property name="text">
         <string>Rectangles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usesLines">
        <property name="text">
         <string>Lines</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottomButtons">
        <item>
         <widget class="QPushButton" name="runStopButton">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stepButton">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
